src/helpers/misc.js

```diff
@@ -116,6 +116,36 @@ const dirCheckSync = (dir) => {
 
 };
 
+/**
+ * Function returns an array that contains unique topics types. Values are taken from argv.topics
+ * -t 'services:serviceOne,tools:toolOne,services:serviceTwo' will return: [ 'services', 'tools' ]
+ * @param  {String} [message] - argv.topics string
+ * @return {Array} - Array with unique topic types
+ */
+function uniqTopicTypes(message) {
+  if (!message) return [];
```

Comment:
Should this return empty array? 
If there is no topic provided should this go on?